.github/workflows/ci.yaml

```diff
@@ -0,0 +1,65 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Continuous Integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - main
```

Comment:
do we need both?

Reply:
Not necessarily. It's a pure convenience sake. Since `main` is the new default, and if you want to set up CI elsewhere, and happen to use this file as a template, you wouldn't need to think if a project there is with `main` or `master`.

Shall we remove?

Reply:
yeah, drop the `master` pls
no value in having it in long run - nobody should use the legacy old default branch name nor use this file from here as template for other repo

Reply:
Aye, done. I thought this is on `master` still.

Reply:
i just changed it 1min before writing the request 3 mins ago :D